tests/behat/marking_summary_double.feature

```diff
@@ -15,45 +15,37 @@ Feature: When a coursework uses double marking the marking summary table should
     Given I log in as the teacher
     And I visit the coursework page
     Then I should see marking summary:
-      | Submitted           | 0/1 |
-      | Needs marking       | 0   |
-      | Marked              | 0   |
-      | Marked and released | 0   |
+      | Submitted         | 0/1 |
+      | Ready for release | 0   |
+      | Marks released    | 0   |
 
   Scenario: Manager's view when there are no submissions
     Given I log in as the manager
     And I visit the coursework page
     Then I should see marking summary:
-      | Submitted            | 0/1 |
-      | Needs marking        | 0   |
-      | Marked (agreed mark) | 0   |
-      | Initial assessor 1   | 0   |
-      | Initial assessor 2   | 0   |
-      | Marked and released  | 0   |
+      | Submitted         | 0/1 |
+      | Ready for release | 0   |
+      | Marks released    | 0   |
 
   Scenario: Teacher's view when there is a submission
     Given the student has a submission
     And the submission is finalised
     And I log in as the teacher
     And I visit the coursework page
     Then I should see marking summary:
-      | Submitted           | 1/1 |
-      | Needs marking       | 1   |
-      | Marked              | 0   |
-      | Marked and released | 0   |
+      | Submitted         | 1/1 |
+      | Ready for release | 0   |
+      | Marks released    | 0   |
 
   Scenario: Manager's view when there is a submission
     Given the student has a submission
     And the submission is finalised
     And I log in as the manager
     And I visit the coursework page
     Then I should see marking summary:
-      | Submitted            | 1/1 |
-      | Needs marking        | 1   |
-      | Marked (agreed mark) | 0   |
-      | Initial assessor 1   | 0   |
-      | Initial assessor 2   | 0   |
-      | Marked and released  | 0   |
+      | Submitted         | 1/1 |
+      | Ready for release | 0   |
+      | Marks released    | 0   |
```

Comment:
not sure about this. marker view changed but not the manager one?

Reply:
They all seemed to change from what I can see.